vueui/src/components/view/late-choices.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div>
+    <div class="clear-both">
+      <vui-img name="character" class="character-image"/>
+      <p>
+        Welcome, <strong>{{ character_name }}</strong>.<br>
+        Round duration: <strong>{{ round_duration }}</strong><br>
+        Alert level: <strong :style="{color: alertLevelColor }">{{ alert_level }}</strong><br>
```

Comment:
I see in the preview pictures you provided that this isn't green when it displays green?

Reply:
Indeed, you can see the `alertLevelColor` function in the bottom. My reasoning was that we want to highlight only "unusual" states, so green is just regular text color.